### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ddl-ahmet-gyger**
Updating the licenses information

### CODE REVIEW

**Comment by ddl-ahmet-gyger** [comment]
@niole  - can you review?

**Comment by ddl-ahmet-gyger** [comment]
https://dominodatalab.atlassian.net/browse/DOM-53273

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## License
-This template is licensed to Customer subject to the terms of the license agreement between Domino and the Customer on file.
+This template is licensed under Apache 2.0 and contains the following open source components: Langchain ([MIT](https://github.com/langchain-ai/langchain/blob/master/LICENSE), Pinecone ([EULA](https://www.pinecone.io/thin-client-user-agreement/), [OpenAI](https://openai.com/policies/terms-of-use), and [FAISS](https://github.com/facebookresearch/faiss/blob/main/LICENSE)).
```
Comment (niole) [inline README.md]: Screen Shot 2023-12-20 at 1 24 13 PM
 the parenthesis look unbalanced

Comment (ddl-ahmet-gyger) [reply README.md]: Good catch, fixed @niole .

**Review by niole — APPROVED** [review]

**Pull request merged**